smarty/data/atomtypes/basetypes.smarts

```diff
@@ -10,3 +10,5 @@
 [#34]   selenium
 [#35]   bromine
 [#53]   iodine
+[*]     any_atom
+[$ewg1] ewg1
```

Comment:
I would put these at the top of the list, that way if someone were to use this file for the initialtypes option the '*' and 'ewg1' would get over written.